kbase-extension/static/kbase/js/widgets/appWidgets2/validators/constants.js

```diff
@@ -0,0 +1,17 @@
+define([], () => {
```

Comment:
This may grow, depending on the strategy picked for merging validator modules. If they all wind up getting merged into a validator monolith, this will get included there and the various inputs will get their imports adjusted.